server/container_create.go

```diff
@@ -611,3 +611,56 @@ func (s *Server) setupSeccomp(ctx context.Context, specgen *generate.Generator,
 	specgen.Config.Linux.Seccomp = linuxSpecs
 	return nil
 }
+
+func (s *Server) setupSeccompForProfileField(
```

Comment:
And this could be simplified to just `setupSeccomp`.

Reply:
Fixed as suggested :+1: